README.md

```diff
@@ -261,7 +261,9 @@ To actively develop or debug this plugin you can either work directly within the
    This repo includes a fully working, self-seeding instance of Payload that installs the plugin directly from the source code. This is the easiest way to get started. To spin up this demo, follow these steps:
 
    1. First clone the repo
-   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp env.example .env && yarn && yarn dev`
+   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp .env.example .env && yarn`
+   1. Start the database - You can use docker `cd YOUR_PLUGIN_REPO/demo && docker compose up` or a locally installed mongodb instance.
+   1. Start payload `cd YOUR_PLUGIN_REPO/demo && yarn dev`
```

Comment:
Need to capitalize `Payload` here but also the user is already in the demo directory, so we can simply run `yarn dev` at this step.

```suggestion
   1. Start Payload `yarn dev`
```